sympy/polys/numberfields.py

```diff
@@ -53,30 +53,27 @@ def _choose_factor(factors, x, v, dom=QQ, prec=200, bound=5):
     if len(factors) == 1:
         return factors[0]
 
-    points = {x:v}
+    factors = {f:f.as_expr().subs({x:v}) for f in factors}
```

Comment:
If `v.is_number` it should be put in with the `points` for best evaluation.

Reply:
Bearing in mind that these are always exact algebraic polynomials is there a benefit in passing them to evalf rather than just using subs?